pkg/commands/workload_update.go

```diff
@@ -184,7 +184,11 @@ func (opts *WorkloadUpdateOptions) IsDryRun() bool {
 }
 
 func NewWorkloadUpdateCommand(ctx context.Context, c *cli.Config) *cobra.Command {
-	opts := &WorkloadUpdateOptions{}
+	opts := &WorkloadUpdateOptions{
+		WorkloadOptions: WorkloadOptions{
```

Comment:
I would recommend adding LoadDefaults func on `WorkloadOptions`. In that func, you can set ExcludeFilePath to the default file. Declare the default path of tanzuignore as const in workload.go file. In this pattern, you can remove TanzuIfnoreFile from cli config completely.  WDYT?

Reply:
I like the idea of `LoadDefaults` function but the `cli.Config`'s `TanzuIfnoreFile` is to be used later as a global cli configuration and i see more sense in cofig besides to kubecofig file and viper config file

Reply:
Added `LoadDefaults` *func* to `WorkloadOptions` and called on each sub-command